corehq/messaging/scheduling/models/content.py

```diff
@@ -0,0 +1,45 @@
+import jsonfield
+from corehq.messaging.scheduling.models.abstract import Content
+from django.db import models
+
+
+class SMSContent(Content):
+    message = jsonfield.JSONField(default=dict)
+
+    def send(self, recipient):
+        print '*******************************'
+        print 'To:', recipient
+        print 'Message: ', self.message
+        print '*******************************'
+
+
+class EmailContent(Content):
+    subject = jsonfield.JSONField(default=dict)
+    message = jsonfield.JSONField(default=dict)
+
+    def send(self, recipient):
+        print '*******************************'
+        print 'To:', recipient
+        print 'Subject: ', self.subject
+        print 'Message: ', self.message
+        print '*******************************'
+
+
+class SMSSurveyContent(Content):
+    form_unique_id = models.CharField(max_length=126)
```

Comment:
I think this would be fine, but flagging that this should be 255 based on https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/models.py#L185

(i just put it on my backlog to make constants for these things and try to standardize these lengths across HQ)